### ISSUE TITLE
Stack Rewrite

### ISSUE THREAD

**Issue by naixx** [halfhp/rxtracer#1]
Shouldn't be the line:
[this](https://github.com/halfhp/rxtracer/blob/master/rxtracer/src/main/java/com/halfhp/rxtracer/RxTracer.java#L97)
```java
System.arraycopy(concatenatedTrace, 0, concatenatedTrace, 0, concatenatedTrace.length);
```

be like this?
```java
System.arraycopy(originalTrace, 0, concatenatedTrace, 0, originalTrace.length);
```

**Comment by halfhp** [comment]
Uggh, I totally missed what you were pointing out here.  Fixed and going up shortly

**Comment by halfhp** [comment]
Fixed and released in 0.1.2
